airbyte-integrations/connectors/source-shortio/source_shortio/manifest.yaml

```diff
@@ -1,106 +1,432 @@
 version: "0.86.0"
 
 definitions:
-  selector:
```

Comment:
I _think_ we've made a change that made it so Builder would ref out blocks like this, so they don't get duplicated in each stream. That's sort of neat. We could always review this later with an automated PR wave, but just curious why this got dereferenced. Did you do that manually, or was that builder?

Reply:
Not actually, Currently Shortio is not supported by the builder, as it uses different base URLs for different streams
So, Just removed parameters macro, embedded schema and spec fo this connector.
![image](https://github.com/airbytehq/airbyte/assets/47392334/22deef97-44b2-46c5-ab74-0202425026df)

Reply:
@natikgadzhi 
Not actually, Currently Shortio is not supported by the builder, as it uses different base URLs for different streams
So, Just removed parameters macro, embedded schema and spec fo this connector.
![image](https://github.com/airbytehq/airbyte/assets/47392334/22deef97-44b2-46c5-ab74-0202425026df)

Reply:
What if we use empty base url and set full url for each stream? ;)